src/cdk/drag-drop/drag-ref.ts

```diff
@@ -297,6 +297,9 @@ export class DragRef<T = any> {
   /** Arbitrary data that can be attached to the drag item. */
   data: T;
 
+  /** Location for preview clone to append into */
+  previewContainer: HTMLElement;
```

Comment:
This should follow the same pattern as the root and boundary elements where we have a method that accepts both `ElementRef` and `HTMLElement`.

Reply:
I was thinking about that, but I was a little apprehensive `ElementRef` has `ElementRef.nativeElement`, and it's already getting coerced to a `HTMLElement` in the `Drag` directive.

I've updated it now, do you think it's worth re-running the `coerceElement` on the `const previewContainer`

Reply:
I thought I commented on it already, but I can't find the comment now so here it is again: instead of having the `previewContainer` be a public property, we should use a `withPreviewContainer` method instead, like we're doing for the root element. That would be a good place to run `coerceElement` as well.

Reply:
done that now :-)

Reply:
done this - will be in next push